ReleaseChecklist.md

```diff
@@ -11,6 +11,7 @@ Checklist for making a release:
  - [ ] Run ``scripts/release_ppa.sh release`` to create the PPA release (you need the relevant openssl key).
  - [ ] Check that the Docker release was pushed to Docker Hub (this still seems to have problems).
  - [ ] Update the homebrew realease in https://github.com/ethereum/homebrew-ethereum/blob/master/solidity.rb (version and hash)
+ - [ ] Update the default version on readthedocs.
```

Comment:
Do you mean to update it for every single example ?

Reply:
No, it's a setting in the UI of readthedocs.

Reply:
Basically the setting is where `solidity.readthedocs.io` points.

Reply:
Ah cool, so far it was set to `develop` and you've fixed it to a released version.